### PR TITLE
Add developer setting to send many messages at once, for debugging purposes

### PR DESCRIPTION
**Pull request by marcoconti83**
## What's new in this PR?

I am investigating possible causes for decryption errors when exchanging messages. I added this setting to automate sending messages back and forth between two devices to see if I can reproduce the problem. I am committing this because I believe this to be useful debugging tool that might come in handy at some other time.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Settings/CellDescriptors/SettingsCellDescriptorFactory.swift [hunk 1]
```diff
@@ -549,6 +557,31 @@ class SettingsCellDescriptorFactory {
         }
     }
     
+    /// Sends a message that will fail to decode on every other device, on the first conversation of the list
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/Settings/CellDescriptors/SettingsCellDescriptorFactory.swift:560]: Why does it fail to decode on every other device?

Comment (marcoconti83) [reply Wire-iOS/Sources/UserInterface/Settings/CellDescriptors/SettingsCellDescriptorFactory.swift:560]: Ops, copy-paste error. This comment is not relevant here

Comment (marcoconti83) [inline Wire-iOS/Sources/UserInterface/Settings/CellDescriptors/SettingsCellDescriptorFactory.swift:560]:
```suggestion
    /// Sends a number of messages to the top conversation in the list, in an asynchronous fashion
```

Comment (marcoconti83) [reply Wire-iOS/Sources/UserInterface/Settings/CellDescriptors/SettingsCellDescriptorFactory.swift]: Fixed!

**Pull request merged**